app-frontend/src/app/pages/projects/detail/detail.html

```diff
@@ -1,117 +1,150 @@
-            {{$ctrl.project.name}}
-            <i class="icon-load" ng-show="$ctrl.isSavingProjectNameEdit"></i>
-          </span>
-        </h1>
-        <button class="btn btn-tiny"
-                ng-click="$ctrl.toggleProjectNameEdit()"
-                ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
-          <i class="icon-pencil"></i>
-        </button>
-        <div class="color-danger" ng-if="$ctrl.showError">
-          There was an error renaming the project.
-          <button class="btn btn-tiny" type="submit"
-                  ng-click="$ctrl.dismissErrorMessage()">
-            <i class="icon-cross color-danger"></i>
-          </button>
-        </div>
-        <div ng-if="$ctrl.isEditingProjectName">
-          <form class="ng-pristine ng-valid">
-            <div class="form-group all-in-one">
-              <input id="name" type="text" class="form-control"
-                     placeholder="Edit project name" ng-model="$ctrl.editedProjectName"
-                     ng-attr-autofocus="{{$ctrl.isEditingProjectName || undefined }}">
-              <button class="btn btn-link" type="submit"
-                      ng-click="$ctrl.cancelProjectNameEdit()"
-                      ng-if="!$ctrl.isSavingProjectNameEdit">
-                <i class="icon-cross"></i>
-              </button>
-              <button class="btn btn-link" type="submit"
-                      ng-click="$ctrl.saveProjectNameEdit()"
-                      ng-if="!$ctrl.isSavingProjectNameEdit">
-                <i class="icon-check"></i>
+  <div class="container dashboard">
+    <div class="main">
+      <div class="row content stack-sm">
+        <div class="column-8" ng-if="!$ctrl.loading">
+
+          <!-- Dashboard Header -->
+          <a class="breadcrumb" ui-sref="projects.list"><i class="icon-arrow-left"></i> Projects</a>
+          <div class="dashboard-header">
+            <button class="btn btn-tiny page-title-edit"
+                    ng-click="$ctrl.toggleProjectNameEdit()"
+                    ng-show="!$ctrl.$parent.selectedScenes.size && !$ctrl.isEditingProjectName && !$ctrl.isSavingProjectNameEdit">
+              <i class="icon-pencil"></i>
+            </button>
+            <h1 class="h3 page-title">
+            <span ng-hide="$ctrl.isEditingProjectName">
+              {{$ctrl.project.name}}
+              <i class="icon-load" ng-show="$ctrl.isSavingProjectNameEdit"></i>
+              </span>
+            </h1>
+            <div class="color-danger" ng-if="$ctrl.showError">
+              There was an error renaming the project.
+              <button class="btn btn-tiny" type="submit"
+                      ng-click="$ctrl.dismissErrorMessage()">
+                <i class="icon-cross color-danger"></i>
               </button>
             </div>
-          </form>
+            <div class="dashboard-header-reveal" ng-hide="!$ctrl.isEditingProjectName">
```

Comment:
The vertical alignment here is a bit uncomfortable.

<img width="202" alt="screen shot 2017-05-09 at 1 48 31 pm" src="https://cloud.githubusercontent.com/assets/2442245/25865033/9058619c-34bf-11e7-8ac4-b0d510c98d99.png">